bun.lock

```diff
@@ -4,17 +4,19 @@
     "": {
       "name": "@ronin/blade",
       "dependencies": {
+        "@ronin/compiler": "0.18.7",
```

Comment:
These cannot be bundled, otherwise their code would be duplicated, since `ronin`, which depends on those packages, is defined as a regular dependency.